{{cookiecutter.project_slug}}/Jenkinsfile

```diff
@@ -4,29 +4,41 @@ pipeline {
         stage("Checkout") {
             steps {
                 checkout scm
+                sh "mkdir -p var/static var/media var/log"
```

Comment:
Ik heb in wat testprojectjes van mezelf voor dit soort regels een `Makefile` geschreven. Met een `make install` worden dan de benodigde dirs aangemaakt (en soms ook pipenv gedraaid). Zoiets:

    install:
    	pipenv install -e .
    	mkdir -p var/log

Je moet het lokaal immers ook doen.

Reply:
Zo'n makefile is inderdaad mooi maar t voegt wel een extra abstractielaag toe. In dit geval vind ik dat het niet waard. Ook verwarrend als je 'make install' doet en dan under the hood 'pipenv install --deploy --dev' aanroept.